docs/admin/kubelet.md

```diff
@@ -84,6 +84,7 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
       --healthz-port=0: The port of the localhost healthz endpoint
   -h, --help=false: help for kubelet
       --host-network-sources="": Comma-separated list of sources from which the Kubelet allows pods to use of host network. For all sources use "*" [default="file"]
+      --host-pid-sources="": Comma-separated list of sources from which the Kubelet allows pods to use the host pid namespace. For all sources use "*" [default="file"]
```

Comment:
Do we need one flag per namespace or can we combine them? Do we have use cases where admins will enable host namespace sharing for individual namespaces?
cc @smarterclayton